src/pages/cluster-management/clusterInfo.html

```diff
@@ -151,6 +151,6 @@ <h4>{{'cluster_management.cluster_configuration.nodes_list_label' | translate}}<
     </div>
 </div>
 
-<div  ng-if="!loader" class="nodes">
+<div  ng-if="!loader" ng-hide="loader" class="nodes">
```

Comment:
Is this actually working? If `loader=false` you don't render it, but if `loader=true` you hide it?

Reply:
Well it works :). I removed it as it was left from something else